<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сигнал</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Напряжение</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="1" column="4">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Кол.</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Прямо</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Стоп</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Шаг</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Старт</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>Обратно</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="4">
                 <widget class="QSpinBox" name="forward_count"/>
                </item>
                <item row="3" column="4">
                 <widget class="QSpinBox" name="backward_count"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="forward_start">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="backward_start">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QLineEdit" name="forward_stop">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="2">
                 <widget class="QLineEdit" name="backward_stop">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="QLineEdit" name="forward_step">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="3">
                 <widget class="QLineEdit" name="backward_step">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="5">
                 <widget class="QCheckBox" name="forward_dec">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="3" column="5">
                 <widget class="QCheckBox" name="backward_dec">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="1" column="5">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Дек.</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Время</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="1" column="0">
                 <widget class="QLineEdit" name="forward_ms">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>мкс</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="forward_mcs">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>мс</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLineEdit" name="backward_ms">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="backward_mcs">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Подача сигнала</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Порядок подачи сигнала:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="direction_combobox">
                  <item>
                   <property name="text">
                    <string>прямо-обратно</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>обратно-прямо</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Повторить:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="repeat_count">
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>раз</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Условие остановки</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="3">
                 <widget class="QLabel" name="shutdown_min_label">
                  <property name="text">
                   <string>мин.</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>условие</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="5">
                 <widget class="QLineEdit" name="shutdown_enc"/>
                </item>
                <item row="2" column="4">
                 <widget class="QLineEdit" name="shutdown_max"/>
                </item>
                <item row="2" column="0">
                 <widget class="QComboBox" name="terminator_combobox">
                  <item>
                   <property name="text">
                    <string>pass</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>==</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>&gt;</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>&lt;</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>&gt;&lt;</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>&lt;&gt;</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="3">
                 <widget class="QLineEdit" name="shutdown_min"/>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="terminator_measure_combobox_label">
                  <property name="text">
                   <string>ед. изм.</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QComboBox" name="terminator_measure_combobox">
                  <item>
                   <property name="text">
                    <string>Ом</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="4">
                 <widget class="QLabel" name="shutdown_max_label">
                  <property name="text">
                   <string>макс.</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="5">
                 <widget class="QLabel" name="shutdown_enc_label">
                  <property name="text">
                   <string>накоп.</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="shutdown_value_label">
                  <property name="text">
                   <string>знач.</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="2">
                 <widget class="QLineEdit" name="shutdown_value"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Просмотр</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="button_graph">
                  <property name="text">
                   <string>График сигнала</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="json_plot_type">
                  <item>
                   <property name="text">
                    <string>stem</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>plot</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Всего запросов к плате:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_count_tasks">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Имя эксперимента:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="json_name">
             <property name="text">
              <string>новый</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_save">
             <property name="text">
              <string>Сохранить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_cancel">
             <property name="text">
              <string>Отмена</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_png">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>hold.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
